### PR TITLE
Allow multiple return values in willReturn

### PR DESCRIPTION
**Pull request by stefk**
Hi,

This PR proposes a shorter alternative to the `willReturnOnConsecutiveCalls` method, using the existing `willReturn`. Example:

```php
$mock->expects($this->any())
    ->method('foo')
    ->willReturn(1, 2, 3);

$mock->foo(); // return 1
$mock->foo(); // return 2
$mock->foo(); // return 3
``` 

It's basically just syntactic sugar, already seen in other mocking frameworks. No backward compatibility issue. I added a test case for it.

### CODE REVIEW

**Comment by sebastianbergmann** [comment]
I do not really want to add new features or make big changes to phpunit-mock-objects anymore. This is more likely to get merged than #218 but I am not sure yet.

File: src/Framework/MockObject/Builder/InvocationMocker.php [hunk 1]
```diff
@@ -81,14 +81,22 @@ public function will(PHPUnit_Framework_MockObject_Stub $stub)
     }
 
     /**
-     * @param  mixed                                                 $value
      * @return PHPUnit_Framework_MockObject_Builder_InvocationMocker
      */
-    public function willReturn($value)
+    public function willReturn()
```
Comment (sebastianbergmann) [inline src/Framework/MockObject/Builder/InvocationMocker.php]: Can you refactor this to use variadic arguments? Thanks!

**Comment by stefk** [comment]
@sebastianbergmann, done. Also changed `willReturnOnConsecutiveCalls` for consistency.

**Comment by sebastianbergmann** [comment]
Merged manually, thanks.

**Pull request closed without merging**